static/sass/styles.scss

```diff
@@ -240,6 +240,9 @@ $color-navigation-background: #252525;
 @import 'snapcraft-p_grid';
 @include snapcraft-p-grid;
 
+@import 'snapcraft_p-featured-snap';
+@include snapcraft-p-featured-snaps;
```

Comment:
Nit picking, but why the inconsistency between file name and mixin (snap vs snaps)?